lib/util.js

```diff
@@ -410,7 +394,7 @@ function formatValue(ctx, value, recurseTimes) {
     });
   }
 
-  var constructor = getConstructorOf(value);
+  var constructor = internalUtil.getConstructorOf(value);
```

Comment:
Can we extract this so we're not referencing `internalUtil` each time? Until lib/internal/util.js gets refactored to use the "new" `module.exports = { ... }` format, the property lookup could be slow.